api_spec.yaml

```diff
@@ -3660,11 +3626,15 @@ paths:
 
         The request must originate from public tenant, and the new tenant will use connectionUriDomain and app from which the request originates.
 
+        Note: Updation of core config is not allowed for the default connectionUriDomain, public app and tenant.
```

Comment:
this will also go in the connectionuri and appId PUT APIs.

Reply:
Done